pkg/apis/v1alpha1/register.go

```diff
@@ -82,6 +82,8 @@ var (
 	ResourceNVIDIAGPU             v1.ResourceName         = "nvidia.com/gpu"
 	ResourceAMDGPU                v1.ResourceName         = "amd.com/gpu"
 	ResourceAWSNeuron             v1.ResourceName         = "aws.amazon.com/neuron"
+	ResourceAWSNeuronDevice       v1.ResourceName         = "aws.amazon.com/neurondevice"
```

Comment:
I think we want to stop making updates to the alpha. I'm assuming that we want to carrot people over to the beta by dropping feature support for these things on the alpha versions